Gemfile.lock

```diff
@@ -1,16 +1,33 @@
 GEM
   remote: https://rubygems.org/
   specs:
+    activesupport (5.2.3)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
```

Comment:
out of curiosity: the spacing difference here do mean something?

Reply:
The Gemfile.lock file is auto-generated based on the Gemfile. 
It seems the indented library names are dependencies for the corresponding unindented items.